src/sass/partials/global/_variables.scss

```diff
@@ -0,0 +1,8 @@
+$font-stack:    HelveticaNeue, Helvetica, Arial, sans-serif;
+
+$white: #fff;
+
+// Breakpoints
+$small: 900px;
```

Comment:
just roughly since we have no comps. if you have preferences for breakpoints I can change it.